src/resultsdisassemblypage.cpp

```diff
@@ -277,8 +277,12 @@ void ResultsDisassemblyPage::setData(const Data::DisassemblyResult &data)
     m_appPath = data.appPath;
     m_extraLibPaths = data.extraLibPaths;
     m_arch = data.arch.trimmed().toLower();
+    m_objdump = data.objdumpPath;
```

Comment:
set it on this page from the mainwindow directly, instead of getting it from the data